lib/components/formik/Radio.js

```diff
@@ -6,13 +6,14 @@ const RadioGroup = ({
   label,
   name,
   options,
-  className,
-  childrenClassName,
-  stacked,
+  className = "",
+  childrenClassName = "",
```

Comment:
Can we rename it to `childClassName`?

Reply:
Sure. Will reflect the change in Radio component too.

Refer: https://github.com/bigbinary/neeto-help-web/blob/d3859334338ac89ef09f6715d0115c9d2c565888/app/javascript/Admin/Components/NitroUI/Radio.js#L12